packages/cdai/.storybook/styles.scss

```diff
@@ -0,0 +1,43 @@
+$css--font-face: true;
```

Comment:
Currently, each component (Security and Experimental) has an SCSS file for both the component and story. We should probably discuss how to proceed.

Reply:
Again, this was just a copy paste from our previous spot